client/src/sections/resources/JuneCleaver.tsx

```diff
@@ -0,0 +1,106 @@
+import { $item, get, have } from "libram";
+import { List, ListItem } from "@chakra-ui/react";
+import Line from "../../components/Line";
+import Tile from "../../components/Tile";
+import { AdviceTooltip } from "../../components/Tooltips";
+
+const fightsLeft = get(`_juneCleaverFightsLeft`);
+const cleaverQueue = get(`juneCleaverQueue`)
+  .split(`,`)
+  .map((value) => parseInt(value));
+const skipsRemaining = 5 - parseInt(get(`_juneCleaverSkips`));
+const choiceMap = [
```

Comment:
would it make more sense for this to be an object keyed by choice id than an array?

Reply:
probably not! but maybe!